odoo/addons/base/models/ir_actions_report.py

```diff
@@ -452,7 +452,7 @@ def _run_wkhtmltopdf(
         try:
             wkhtmltopdf = [_get_wkhtmltopdf_bin()] + command_args + files_command_args + paths + [pdf_report_path]
             process = subprocess.Popen(wkhtmltopdf, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-            out, err = process.communicate()
+            out, err = process.communicate(timeout=float(config['limit_time_real']))
```

Comment:
I think it would be better for this to have its own config parameter, since this should be much lower than the normal timeout. Something like limit_time_pdf

Reply:
Hello,
A `limit_time_pdf` seems better and could have a lower value in your case.
It just mitigates the issue, but to solve it we will have to move PDF generation outside of the http workers.